source/ed/ed2nav.cpp

```diff
@@ -165,6 +165,33 @@ struct FindAdminWithCities {
     }
 };
 
+bool write_data_to_file(const std::string& output_filename, navitia::type::Data& data) {
+    auto logger = log4cplus::Logger::getInstance("log");
+    std::string temp_output = output_filename + ".temp";
+    try {
+        data.save(temp_output);
+        try {
+            remove(output_filename.c_str());
```

Comment:
If this is `remove` from `stdio` you need to check the return value, same things for `rename`.